main.js

```diff
@@ -11,5 +16,13 @@ function handleMove(event) {
     $racecar.className = 'racecar up';
   } else if (event.key === 'ArrowDown') {
     $racecar.className = 'racecar down';
+  } else if (event.key === ' ') {
+    setInterval(moveCar, 16);
   }
 }
+
+function moveCar() {
+  data.x += 10;
+  var position = 'top: ' + data.y + 'px; left: ' + data.x + 'px;';
```

Comment:
You can do this directly in the JS by assigning a value to `$racecar.style.left` and ``$racecar.style.top`